dom.bs

```diff
@@ -6929,7 +6931,11 @@ are:
 <div algorithm>
 <p>To <dfn id=concept-attach-a-shadow-root>attach a shadow root</dfn>, given an
 <a for=/>element</a> <var>element</var>, a string <var>mode</var>, a boolean <var>clonable</var>,
-a boolean <var>delegatesFocus</var>, and a boolean <var>slotAssignment</var>:
+a boolean <var>delegatesFocus</var>, and a string <var>slotAssignment</var>:
```

Comment:
Note: I just fixed up the `slotAssignment` type, which was incorrectly listed as a `boolean` before.